nixos/modules/services/web-apps/siyuan.nix

```diff
@@ -0,0 +1,79 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.siyuan;
+  username = "siyuan";
+  workingDirectory = "/var/lib/siyuan";
+in
+
+{
+  options.services.siyuan = {
+    enable = mkEnableOption "the Siyuan service";
+
+    package = mkPackageOption pkgs "siyuan" { };
+
+    language = mkOption {
+      type = types.enum [ "zh_CN" "zh_CHT" "en_US" "fr_FR" "es_ES" ];
+      default = "en_US";
+      description = "The language Siyan should use.";
+    };
+
+    listenHost = mkOption {
```

Comment:
Is this used anywhere?

Reply:
Ah, no, indeed. I expected this to be parameterizable when calling `SiYuan-Kernel`, but I don't see anything...
Am I missing anything, or can it only run on `localhost` (the default)?

Reply:
I don't know, but I found this: https://github.com/siyuan-note/siyuan/blob/3eec016723f52f3a679fb99854ae33e8fe20e42a/kernel/server/serve.go#L104C23-L108

Somehow networkServe can be toggled, though I don't exactly know how.

Reply:
Maybe this: https://github.com/siyuan-note/siyuan/blob/3eec016723f52f3a679fb99854ae33e8fe20e42a/app/src/config/about.ts#L318-L326